numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -928,25 +928,24 @@ def add_newdoc(place, name, doc):
 
     The Heaviside step function is defined as::
 
-                             0   if x < 0
-        heaviside(x, h0) =  h0   if x == 0
-                             1   if x > 0
```

Comment:
The source of this double $PARAMS$ is a careless find-and-replace by me. I think the cause of the missing one is the addition of `isnat` being merged after #9026, without the PR being updated